packages/components/ng-at-internet/src/constants.js

```diff
@@ -49,6 +49,14 @@ export const AT_INTERNET_CUSTOM_VARS = {
     path: 'page.2', // OVH's AtInternet configuration
     format: '[%s]',
   },
+
+  /**
+   * Voucher code used
```

Comment:
Looks like a copy/paste from the constant above.
Could you please update it to reflect the meaning of this constant?

In addition, maybe you can split your commit in two distinct commit.
The first one act more as a feature (adding a new constant)
and then the second one is fixing the order tracking component.

Is that make sense?